static/locales/en-US.yaml

```diff
@@ -317,6 +317,11 @@ Settings:
     Fetch Automatically: Fetch Feed Automatically
   Distraction Free Settings:
     Distraction Free Settings: Distraction Free Settings
+    Sections:
+      Side Bar: Side Bar
+      Channel Pages: Channel Pages
```

Comment:
Channels? Channel Views?
I am not sure if we should call it "Pages"

Reply:
After some thought I can go with `Channel Page` or `Channel View`
No `s` (that's the point that makes me feel weird)

Reply:
Decided to go with page instead of view, as we use page in other strings too.